### ISSUE TITLE
Version detection broken

### ISSUE THREAD

**Issue by itsrachelfish** [OpenKore/openkore#2]
Because of the migration to git, the automatic version detection no longer works:

```
*** OpenKore what-will-become-2.1 ( r? ) - Custom Ragnarok Online client ***
***   http://www.openkore.com/   ***

Selectively loading plugins...
```

We'll need to update the version detection code to use git and ideally provide a partial commit hash as well as a date.

**Comment by ChrstphrR** [comment]
More historical than helpful:

The version detection was broken in the SVN repository, which just didn't show up for most, unless you had a more recent subversion client version, and checked out a new copy of the repository.

allanon has a code snippet he was fleshing out, that will detect the commit hash, so a fix may be pending already.

**Comment by itsrachelfish** [comment]
Fixed as of pull request #6 (commit 03cd8bd)
